cookbooks/passenger5/files/default/passenger_monitor

```diff
@@ -70,7 +71,7 @@ for record in `passenger-memory-stats | egrep -i "Passenger RubyApp:" | sed -r '
   ## Check the against the limit, if it exceeds the memory amount, KILL IT.
   if [[ $app_real -gt $limit ]]; then
     logger -t passenger_monitor -s "Killing PID $app_pid (app $app_name) - memory $app_real MB exceeds $limit MB"
-    kill -USR1 $app_pid
+    ${passenger_utils_bin_path}/passenger-config detach-process $app_pid
```

Comment:
If I'm not mistaken `passenger-config` and `passenger-status` are already in the path. Is there are reason why we prefer using the absolute path instead of just using the passenger-config and passenger-status that's in the path? 

IMO avoiding use of the absolute path will allow us to avoid issues like
https://github.com/engineyard/ey-cookbooks-stable-v5/pull/299

Reply:
As I see $PATH is correct only for user deploy. 
But we are running this command as a cron job. And for cron jobs default path is set to:
PATH=/bin:/usr/bin:/usr/local/bin:/usr/local/ey_resin/bin
The problem is that inside the /usr/bin we have another passenger-config binary which provided by nginx package and which is not working properly. 
The solution would be changing default path for the cron jobs, but we probably will have another problems.

Reply:
@jegasega good points, and thanks for explaining. This looks good to me - please feel free to go ahead, merge and test.